plugins/inputs/docker/docker.go

```diff
@@ -434,7 +434,7 @@ func (d *Docker) gatherContainer(
 	var cname string
 	for _, name := range container.Names {
 		trimmedName := strings.TrimPrefix(name, "/")
-		if len(strings.Split(trimmedName, "/")) == 1 {
+		if !strings.Contains(trimmedName, "/") {
```

Comment:
What about `/name/` should that match? Maybe it would be better to trim both leading _and_ trailing slashed?

Reply:
I don't think you can get a name like that.

I'd prefer to keep this implementation as similar as possible to the Docker CLI implementation. I imagine that users expect the implementation of the name filter to be consistent with the output of `docker ps`.

Reply:
Ok than that's fine with me.